example/src/App.tsx

```diff
@@ -6,951 +6,10 @@ import { GRPCSelfGeneratedAPIReference } from 'grpc-docs'
 const App = () => {
   // return <GRPCGenDocuAPIReference project='LibraryApp' organization='gendocu' />
   // return <GRPCGenDocuAPIReference project='GendocuPublicApis' organization='gendocu' />
+
   return (
-    <GRPCSelfGeneratedAPIReference
```

Comment:
Having an example of the `definition` param would be great. Can you add a commented-out line with the example of usage? And please extract the definition value to some constant in another file in `example/src/`.